tests/c/keep_includes-d1.h

```diff
@@ -0,0 +1,11 @@
+//===--- keep_includes-d1.h - test header for iwyu ----------------------------------===//
```

Comment:
Line runs too long now that the filename is longer, trim dashes to hit 80, please